optuna/visualization/matplotlib/_parallel_coordinate.py

```diff
@@ -66,7 +84,14 @@ def _get_parallel_coordinate_plot(study: Study, params: Optional[List[str]] = No
         all_params = set(params)
     sorted_params = sorted(list(all_params))
 
-    obj_org = [t.value for t in trials if t.value is not None]
+    if target is None:
+
+        def _target(t: FrozenTrial) -> float:
+            return cast(float, t.value)
+
+        target = _target
+
+    obj_org = [target(t) for t in trials]
```

Comment:
[Just a note] Before the change, it checked whether `t.value` was None or not, so the logic seems to be a little different after the change. However, the trials are filtered by `TrialState.COMPLETE`, `t.value` is guaranteed not to be None. There is no behavior change.
```